.gitignore

```diff
@@ -15,6 +15,7 @@ pip-log.txt
 .idea
 .eggs
 venv
+.venv
```

Comment:
hmm, where does this venv come from

Reply:
It was created when I ran `make test`

Reply:
oh, ok 👍 just saw that the entire .gitignore is preemptive anyway